src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.cpp

```diff
@@ -2135,9 +2135,6 @@ void C2_MacroAssembler::enc_cmove(int cmpFlag, Register op1, Register op2, Regis
 // Set dst to NaN if any NaN input.
 void C2_MacroAssembler::minmax_fp(FloatRegister dst, FloatRegister src1, FloatRegister src2,
                                   bool is_double, bool is_min) {
-  assert_different_registers(dst, src1);
-  assert_different_registers(dst, src2);
-
```

Comment:
Ah, Seems I missed this in my last PR :-)  We should remove the `TEMP_DEF dst` effect from the related match rule in riscv.ad file where it is called.
One example: https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/riscv/riscv.ad#L7290

Reply:
Yes, will do it.

Reply:
Thanks. But I think you should keep `KILL cr` effect as this assember routine modifies `t1` thus kills `cr`.

Reply:
Ah, I missed this, thanks for catching!